File: java/types/strings/Strings.java
class Strings {
  public static void main(String[] args) {
    String word = "hello";
    System.out.println(String.format(
      "word = %s (%s)\n"
      + "length: %d\n",
      word,
      word.getClass().getSimpleName(),
      word.length()
    ));

    String cite = "Stat rosa pristina nomine, nomina nuda tenemus.";
    System.out.println(String.format(
      "cite = %s\n"
      + "words count: %d\n",
      cite,
      cite.split("\\s+").length
    ));

    String s1 = "abc";
    String s2 = "def";
    String s3 = s1 + s2;
    System.out.println(String.format("%s + %s = %s\n", s1, s2, s3));
  }
}

File: c#/types/strings/Strings.cs
class Strings {
  public static void Main(string[] args) {
    string word = "hello";
    System.Console.WriteLine(string.Format(
      "word = {0} ({1})\n"
      + "length: {2}\n",
      word,
      word.GetType(),
      word.Length
    ));

    string cite = "Stat rosa pristina nomine, nomina nuda tenemus.";
    System.Console.WriteLine(string.Format(
      "cite = {0}\n"
      + "words count: {1}\n",
      cite,
      cite.Split(' ').Length
    ));

    string s1 = "abc";
    string s2 = "def";
    string s3 = s1 + s2;
    System.Console.WriteLine(string.Format("{0} + {1} = {2}\n", s1, s2, s3));
  }
}
